### PR TITLE
Allow less crawling

### PR DESCRIPTION
**Pull request by ninewise**
There seems to be no robots.txt standard, though, so I've no idea if this works.

### CODE REVIEW

File: content/robots.txt [hunk 1]
```diff
@@ -1,4 +1,15 @@
 User-agent: *
-Allow: /
-Disallow: /wiki/
-Disallow: /slackintegrations
+Disallow: /
```
Comment (niknetniko) [inline content/robots.txt]: Won't this exclude the home page from crawling?

Comment (ninewise) [reply content/robots.txt]: Yes. Is there anything on the home page that's not repeated on other pages (except gamification, which we don't want crawled anyway)?

Comment (redfast00) [reply content/robots.txt]: Well, the homepage itself is quite useful as a landing page (if we exclude it, search engines will give another page when searching for "Zeus WPI")

Comment (ninewise) [reply content/robots.txt]: True, I allowed the home page again in #75c8e6a.

Comment (redfast00) [reply content/robots.txt]: It would be nice if the root itself was also included (I think this is possible with the end-of-path terminator ($))

Comment (niknetniko) [reply content/robots.txt]: Relevant the "$" is documented in the Google robots.txt spec: https://developers.google.com/search/reference/robots_txt

Comment (ninewise) [reply content/robots.txt]: I'm not quite visiting that page, but I'm pretty sure this is the right format. I do hope other crawlers understand this, too.

Comment (redfast00) [reply content/robots.txt]: Maybe also allow sponsorende leden, so websites of sponsorende leden get better pagerank index (from ugent subdomain)

**Pull request merged**